src/lib/statistics/statistics_objects/histogram_domain.cpp

```diff
@@ -0,0 +1,133 @@
+#include "histogram_domain.hpp"
+
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+HistogramDomain<pmr_string>::HistogramDomain()
+    :  // Support most of ASCII with maximum prefix length for number of characters.
+      HistogramDomain<pmr_string>(' ', '~', 9) {}
```

Comment:
Adjusted comment to
```c++
    :  // Support most of ASCII with maximum prefix length for number of characters. The character range and the prefix
      // length were chosen so that the entire range of IntegralType is covered
```